make_ioc_from_project.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+PROJECT_PATH=$1
+
+if [ -z "$(which tcparse-stcmd)" ]; then
```

Comment:
```suggestion
if [ -x "$(command -v tcparse-stcmd)" ]; then
```
In the past I've been told by stackoverflow to do this instead of using `which` for checking if a command exists

Reply:
Thanks!